tests/test_utils.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+from augur.utils import ambiguous_date_to_date_range
+
+from freezegun import freeze_time
+
+
+class TestUtils:
+    def test_abmiguous_date_to_date_range_not_ambiguous(self):
```

Comment:
Minor typos: `test_abmiguous_` should be `test_ambiguous` here and in the next three functions.